src/__tests__/utils/date.test.ts

```diff
@@ -83,15 +82,16 @@ describe('Date methods tests', () => {
     expect(getCurrentYear()).toBe(2020)
   })
 
-  it('parseDate', async () => {
-    MockDate.set(1588694400000, 480)
-    expect(parseDate(1588694380000)).toBe('20s ago')
-    expect(parseDate(1588691000000)).toBe('56min 40s ago')
+  // TODO: not testable, need to pass in a i18n instance
```

Comment:
What is the specific issue here? Can't we get an i18n instance directly during the testing phase?

Reply:
The time format can be done by day.js which has been introduced at https://github.com/Magickbase/ckb-explorer-frontend/blob/develop/package.json#L17

Ref:
- relative time: https://day.js.org/docs/en/customization/relative-time#docsNav
- i18n: https://day.js.org/docs/en/i18n/i18n

Reply:
I tried to utilize relativeTime plugin, but it's not fullly configurable thus it's hard to meet current business requirements, for example, it has a built in rounding method witch can't be disabled by config, rounding `56min 20s` to `57min`, commit is: [d3cba4c](https://github.com/Magickbase/ckb-explorer-frontend/pull/116/commits/d3cba4cfb87f6f90f54aa46e5ef4a5caf9f7a072) then I reverted it